### DISCUSSION TITLE
[1.18.2] Farmers and Tomato Seeds

### DISCUSSION THREAD

**Discussion by LIPDProductionsInc** [vectorwing/FarmersDelight#998] [Q&A]
I'm aware 1.18.2 isn't supported anymore, which is why I'm not putting this as an issue, but is there a way to have the vanilla villager farmer be able to plant tomato seeds after harvesting the tomato crop? I noticed that they don't seem to replace the crop, they just break it.

I would figure this out myself, but I'm not familiar with how coding in Minecraft works

**Comment by Aceplante** [comment]
Theres a mod called smarter farmers; not sure if it would work but you could try that.
